tests/unit/components/PageStepper.spec.js

```diff
@@ -58,3 +58,71 @@ describe('PageStepper getCypressValue()', () => {
     expect(wrapper.find("[data-cy=potato1]").exists()).toBe(true)
   });
 });
+
+describe('a11y', () => {
```

Comment:
One more small nitpick, if that's alright: The name "a11y" isn't very specific. Would you be alright renaming this describe block to something more descriptive, please?

Reply:
Good call, definitely I'll improve the description